### ISSUE TITLE
The trashcan control and the trashcan flyout.

### ISSUE THREAD

**Issue by BenHenning** [google/blockly#9041]

**Comment by BenHenning** [comment]
Trashcan flyout doesn't work as-is today, even with additional work in the keyboard navigation plugin to support it as a proper state. Without digging in more detail I suspect that this may require changes to `WorkspaceSvg` and `Flyout` base to support the case of the flyout being part of the trash can experience. Separately, there may be some work needed for navigation to work here (but I actually expect that to "just work" once the focus bits are corrected).

There's also some oddity in navigating to the trashcan flyout when it's open, but I think that's largely because it's not closing/auto-closing as expected with keyboard navigation. I suspect we'll largely want to treat this similar to the toolbox flyout in how and when it auto-hides.

**Comment by BenHenning** [comment]
Updating the issue to include making the trashcan and its flyout workspace navigable (including navigating into/out of the trashcan flyout).

**Comment by rachel-fenichel** [comment]
@gonfunko does the trash flyout work after your flyout changes?

**Comment by gonfunko** [comment]
It does!

**Comment by rachel-fenichel** [comment]
And is there a way for a user to get into and out of that flyout with keyboard navigation?

**Comment by gonfunko** [comment]
Not entirely, you have to use the mouse to click the trash to open it. It is tabbable once you've done that though, and you can insert from it to restore a block from the trash.
